src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] valuesFromFile = readFromFile(fromFileName);
+        writeInFile(toFileName, valuesFromFile[0], valuesFromFile[1]);
```

Comment:
```suggestion
        String dataFromFile = readFromFile(fromFileName);
        String report = createReport(dataFromFile);
        writeInFile(toFileName, report);
```
lets divide into independent methods, divide functionality by responsibility - read data, parse data, write report